mfs_test.go

```diff
@@ -1182,3 +1182,40 @@ func TestTruncateAndWrite(t *testing.T) {
 		}
 	}
 }
+
+func TestIsDir(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	ds, rt := setupRoot(ctx, t)
+
+	dir := rt.GetDirectory()
+
+	nd := dag.NodeWithData(ft.FolderPBData())
+	di, err := NewDirectory(ctx, "test", nd, dir, ds)
+	if err != nil {
+		t.Fatal(err)
+	}
+	ret := IsDir(di)
+	if !ret {
+		t.Fatal("FSNode type should be dir, but not")
+	}
+}
+
+func TestIsFile(t *testing.T) {
```

Comment:
nit: could we merge this two test functions? or include a helper test function that would abstract all the boiler plate code of creating an MFS to experiment with? (I feel like this should exist already somewhere in this file.)

Reply:
@schomatis I had merged two test function. However, I think it is not a completed job.